modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -123,6 +124,36 @@ public void testGetObjectDefinition() throws Exception {
 			objectDefinition.getRestContextPath());
 	}
 
+	@Override
+	@Test
+	public void testGetObjectDefinitionsPage() throws Exception {
+
+		// super.testGetObjectDefinitionsPage();
+
+		ObjectDefinition objectDefinition =
+			testGetObjectDefinitionsPage_addObjectDefinition(
+				randomObjectDefinition());
+
+		testGetObjectDefinitionsPage_addObjectDefinition(
+			_randomModifiableSystemObjectDefinition());
```

Comment:
I see in the `testPostObjectDefinition` the following code:

```java
	public void testPostObjectDefinition() throws Exception {
		super.testPostObjectDefinition();

		ObjectDefinition randomObjectDefinition = randomObjectDefinition();

		Status status = new Status() {
			{
				code = WorkflowConstants.STATUS_APPROVED;
				label = WorkflowConstants.getStatusLabel(
					WorkflowConstants.STATUS_APPROVED);
				label_i18n = _language.get(
					LanguageResources.getResourceBundle(
						LocaleUtil.getDefault()),
					WorkflowConstants.getStatusLabel(
						WorkflowConstants.STATUS_APPROVED));
			}
		};

		randomObjectDefinition.setStatus(status);
```

I think we should keep the same format, so instead of using the `_randomModifiableSystemObjectDefinition`, we could create a random object definition and set the status to approved

Reply:
Make sense! :+1: